README.md

```diff
@@ -68,23 +68,24 @@ Building a .deb package is pretty similar to build a .rpm package. You need to d
       Usage: ./generate_debian_package.sh [OPTIONS]
 
           -b, --branch <branch>     [Required] Select Git branch or tag e.g.
-          -d, --destination <path>  [Required] Set the destination path of package.
+          -s, --store <path>        [Required] Set the destination path of package.
           -t, --target <target>     [Required] Target package to build [manager/api/agent].
           -a, --architecture <arch> [Required] Target architecture of the package [amd64/i386].
           -r, --revision <rev>      [Required] Package revision that append to version e.g. x.x.x-rev
           -j, --jobs <number>       [Optional] Number of parallel jobs when compiling.
           -p, --path <path>         [Optional] Installation path for the package. By default: /var/ossec.
+          -d, --debug               [Optional] Build the binaries with debug symbols. By default: no.
           -h, --help                Show this help.
 
       ```
     * To build a wazuh-manager package in /tmp for amd64 (x86_64):
-        `# ./generate_debian_package.sh -b 3.4 -d /tmp -t manager -a amd64 -r 0`.
+        `# ./generate_debian_package.sh -b 3.9 -d /tmp -t manager -a amd64 -r 0`.
     * To build a wazuh-agent package in `/home/ec2-user` for i386 with release 2:
-        `# ./generate_debian_package.sh -b v3.5.0 -d /home/ec2-user -t agent -a i386 -r 2`.
+        `# ./generate_debian_package.sh -b v3.8.2 -d /home/ec2-user -t agent -a i386 -r 2`.
     * To build a wazuh-api package in /tmp from branch 3.6:
```

Comment:
The same case mentioned previously.

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/139/commits/eea92e2c82c08bcc398dee0c672de074c8323480.